libs/s25main/factories/JobFactory.cpp

```diff
@@ -179,6 +182,15 @@ std::unique_ptr<noFigure> JobFactory::CreateJob(const Job job_id, const MapPoint
         case Job::TempleServant:
             RTTR_Assert(dynamic_cast<nobUsual*>(goal));
             return std::make_unique<nofTempleServant>(pt, player, static_cast<nobUsual*>(goal));
+        case Job::Skinner:
+            RTTR_Assert(dynamic_cast<nobUsual*>(goal));
```

Comment:
all that C&P...
Maybe a function for the assert-dynamic_cast-then-static_cast would be useful? Can't think of a good name though